src/base/base.scss

```diff
@@ -76,10 +76,43 @@ button {
 }
 
 details {
-  summary { cursor: pointer; }
+  summary {
+    cursor: pointer;
+  }
 
   &:not([open]) {
     // Set details content hidden by default for browsers that don't do this
-    > *:not(summary) { display: none !important; }
+    > *:not(summary) {
+      display: none !important;
+    }
+  }
+}
+
+// global focus styles
+
+// fallback :focus state
+:focus,
+.focus {
+  &:is(a, button, input, textarea, select) {
+    @include focusOutline;
+  }
+}
+
+// remove fallback :focus if :focus-visible is supported
+:focus:is(a, button, input, textarea, select):not(:focus-visible) {
+  outline: solid 1px transparent;
+}
+
+// default :focus-visibile state
+:focus-visible:is(a, button, input, textarea, select) {
+  @include focusOutline;
+}
+
+// remove offset for form fields
+:focus,
+:focus-visible {
+  &:is(input, textarea, select) {
+    border-color: transparent;
```

Comment:
Focused [`form-group` inputs](https://primer-css-dbkqalvz4-primer.vercel.app/css/components/forms#error) still show the border color because of the higher specificity (`.form-group.errored .form-control`). But maybe that should be changed in `form-group.scss` and not `base.scss`.

![Screen Shot 2021-11-11 at 17 46 10](https://user-images.githubusercontent.com/378023/141267602-8780f748-af04-490b-ae9c-315d9a655b1c.png)

Also, it's only really visible when zoomed in, but there is that inset shadow that we might also wanna remove:

![Screen Shot 2021-11-11 at 17 40 33](https://user-images.githubusercontent.com/378023/141266098-2d0190fe-2bc8-4e15-bd4a-22d27189e625.png)

Reply:
Thanks! I addressed it in `form-group`. Before I had addressed it globally, and that's why inputs were disappearing 👻 let me know if you think of other scenarios similar to this..